src/api/app/assets/stylesheets/webui/timeline.scss

```
@@ -15,5 +15,24 @@
         .timeline-item-comment {
             margin-left: $avatars-counter-size + $timeline-border-width + $timeline-item-avatar-margin;
         }
+
+        i.timeline-break, i.timeline-offset {
+            @extend .pt-2;
+            @extend .mb-4;
+            @extend .pb-2;
+            @extend .fas;
+            @extend .fa-ellipsis-vertical;
+            margin-left: 0.66rem;
+        }
+        i.timeline-break {
+            @extend .bg-white;
+            @extend .text-dark;
+        }
+        a:hover i.timeline-break {
+          filter: none!important;
```

A space is missing:

```suggestion
          filter: none !important;
```

We should also explain in a comment why `!important` is needed here and above in the file _collapse-component.scss_.

---

It was not needed. I removed it.